fs/fs.go

```diff
@@ -58,12 +60,15 @@ type RealFsInfo struct {
 	labels map[string]string
 
 	dmsetup dmsetupClient
+
+	thinPoolWatcher *volume.ThinPoolWatcher
```

Comment:
Device mapper outside of docker lacks context. How do you envision users
consuming the device mapper data?
If cAdvisor can support desktop sandbox containers for example which uses
device mapper outside of the docker context, then we can extend device
mapper support to such a plugin.

On Mon, Apr 18, 2016 at 9:25 AM, Paul Morie <notifications@github.com>
wrote:

> In fs/fs.go
> <https://github.com/google/cadvisor/pull/1204#discussion_r60089023>:
>
> > @@ -58,12 +60,15 @@ type RealFsInfo struct {
> >  	labels map[string]string
> >
> >  	dmsetup dmsetupClient
> > +
> > +	thinPoolWatcher *volume.ThinPoolWatcher
>
> @vishh <https://github.com/vishh> also -- is there a reason to keep
> devicemapper as a docker-only notion?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/google/cadvisor/pull/1204/files/ae32fd2351c70119c092810e6d3265706f691b0a#r60089023>
>